docs/api/api-spec.md

```diff
@@ -142,18 +142,41 @@ Context channels allows a set of apps to share a stateful piece of data between
 
 There are two types of channels, which are functionally identical, but have different visibility and discoverability semantics.  
 
-1. The 'system' ones, which have a well understood identity. One is called 'default'.
+1. The 'system' ones, which have a well understood identity. One is called 'global'.
 2. The 'app' ones, which have a transient identity and need to be revealed
-
-The 'system' channels include a 'default' channel which serves as the backwards compatible layer with the 'send/broadcast context' above. There are some reserved channel names for future use. Currently this is just 'global'.
+ 
 
 ### Joining Channels
-Apps can join channels.  An app can only be joined to one channel at a time.  When an app joins a channel it will automatically recieve the current context for that channel, except if the channel joined is the 'default'.
+Apps can join channels.  An app can only be joined to one channel at a time.  When an app joins a channel it will automatically recieve the current context for that channel.
 
-When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not explicitly joined to a channel, it is on the 'default' channel.  It is up to the desktop agent to determine the behavior of the 'default' channel, and if context is automaticaly broadcast over it or not.
+When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not joined to a channel these methods will be no-ops, but apps can still choose to listen and broadcast to specific channels via the methods on the `Channel` class.  
 
 It is possible that a call to join a channel could be rejected.  If for example, the desktop agent wanted to implement controls around what data apps can access.  
 
+Joining channels in FDC3 is intended to be a behavior initiated by the end user. For example: by color linking or apps being grouped in the same workspace.  Most of the time, it is expected that apps will be joined to a channel by mechanisms outside of the app.  Always, there SHOULD be a clear UX indicator of what channel an app is joined to. 
+
+### The 'global' Channel
+The 'system' channels include a 'global' channel which serves as the backwards compatible layer with the 'send/broadcast context' behavior in FDC3 1.0.  A desktop agent MAY choose to make membership in the 'global' channel the default state for apps on start up.  
```

Comment:
👏

Reply:
> ... apps will be joined to a channel by mechanisms outside of the app.

What APIs would be used to do this? All proposed, `.joinChannel(...)`, `.leaveCurrentChannel()`, and `.getCurrentChannel()` APIs are done from within the calling context; e.g. via the local `fdc3: DesktopAgent` reference.

If some external color assignment widget wanted to assign "App A" to channel "red", its not exactly sure how the proposed APIs facilitate this; i.e., there is no API provided modify channel assignment from "outside the app".

Whats missing is something like `fdc3.getApp("App A").joinChannel("red")`...

In lieu of addressing this in the API now, I suggest it might be easier to "say less" and remove the sentence "Most of the time..."

Reply:
Thanks @nicholasgoodman - this raises an excellent point.  The API defined here is for apps to interact with FDC3.  This does not cover what APIs a desktop agent might use to create default behavior for apps wrt to channels.  This omission is purposeful, since this kind of join functionality would be implemented with APIs *internal* to the desktop agent.